metro-ai-suite/metro-vision-ai-app-recipe/compose-scenescape.yml

```
@@ -208,67 +208,46 @@ services:
     restart: on-failure:5
 
   pgserver:
-    image: docker.io/intel/scenescape-manager:v1.4.0
-    init: true
+    image: postgres:17.6
+    env_file:
+      - ./${SAMPLE_APP}/src/secrets/pgserver/pgserver.env
     networks:
       - scenescape
     environment:
-      - "DBROOT=/workspace"
-      - "EXAMPLEDB=smart-intersection-ri.tar.bz2"
-      - http_proxy=${http_proxy}
-      - https_proxy=${https_proxy}
-      - no_proxy=${no_proxy},.scenescape.intel.com,influxdb2
-      - NO_PROXY=${no_proxy},.scenescape.intel.com,influxdb2
-      - HTTP_PROXY=${http_proxy}
-      - HTTPS_PROXY=${https_proxy}
-    entrypoint: [
-        "/bin/bash",
-        "-e",
-        "-c",
-        "cp /tmp/user_access_config.json /home/scenescape/SceneScape/user_access_config.json && \
-        cp /tmp/smart-intersection-ri.tar.bz2 /home/scenescape/SceneScape/smart-intersection-ri.tar.bz2 && \
-        chown -R scenescape:scenescape /workspace && \
-        SUPASS=$(cat /run/secrets/supass) /usr/local/bin/scenescape-init database --preloadexample",
-      ]
-    cap_add:
-      - SYS_ADMIN
-    devices:
-      - /dev/fuse
-    security_opt:
-      - apparmor:unconfined
+      POSTGRES_USER: scenescape
+      POSTGRES_DB: scenescape
     volumes:
-      - pgserver-db:/workspace/db
-      - pgserver-migrations:/workspace/migrations
-      - pgserver-media:/workspace/media
-      - ./${SAMPLE_APP}/src/webserver/user_access_config.json:/tmp/user_access_config.json
-      - ./${SAMPLE_APP}/src/webserver/smart-intersection-ri.tar.bz2:/tmp/smart-intersection-ri.tar.bz2
-    secrets:
-      - django
-      - browser.auth
-      - controller.auth
-      - supass
-    restart: on-failure:5
+      - pgserver-db:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
 
   web:
-    image: docker.io/intel/scenescape-manager:v1.4.0
+    image: ghcr.io/open-edge-platform/edge-ai-libraries/intel/scenescape-manager:2025.2-rc2
     init: true
     networks:
       scenescape:
         aliases:
           - web.scenescape.intel.com
     depends_on:
-      - pgserver
+      pgserver:
+        condition: service_started
+      broker:
+        condition: service_started
     environment:
-      - "DBROOT=/workspace"
       - http_proxy=${http_proxy}
       - https_proxy=${https_proxy}
       - no_proxy=${no_proxy},.scenescape.intel.com,influxdb2
       - NO_PROXY=${no_proxy},.scenescape.intel.com,influxdb2
       - HTTP_PROXY=${http_proxy}
       - HTTPS_PROXY=${https_proxy}
+      - SUPASS
```

```suggestion
      - SUPASS=${SUPASS}
```

The creation of this environment variable either needs to be moved to install script where `.env` file gets populated with this value or documentation needs to be updated saying user needs to export `supass` with a value.

I believe the creation of `/src/secrets/supass` is no longer necessary as this file is no longer supplied as a secret anywhere.
Previously it was supplied as a secret to manager and the createDB init was run with the value being read from the file

```bash
SUPASS=$(cat /run/secrets/supass) /usr/local/bin/scenescape-init database --preloadexample
```
but this is not the case anymore of how it's done